src/app/core/default-pattern-renderer/default-pattern-renderer.component.scss

```diff
@@ -7,4 +7,7 @@
   margin: 0 7em;
   background-color: whitesmoke;
 }
+.button {
+  font-size: 1px;
+}
 
```

Comment:
are you sure this shouldn't be just button instead of .button? . indicates this a custom class (which is not the case for <button> )